string object
string property
string instance
bool data
---
bool result
